hosts/nix-nextcloud/configuration.nix

```
@@ -107,6 +115,34 @@
         "traefik.http.routers.whiteboard.middlewares" = "strip-whiteboard";
       };
     };
+    "appapi-harp" = {
+      image = "ghcr.io/nextcloud/nextcloud-appapi-harp:release";
+      environment = {
+        "HP_SHARED_KEY" = "some_very_secure_password";
+        "NC_INSTANCE_URL" = "https://nextcloud.sotga.de";
+      };
+      volumes = [
+        #"/certs:/certs:rw" #TODO
+        "/var/run/docker.sock:/var/run/docker.sock:rw"
```

In Nixos ist der Docker Socket unter /run/docker.sock

```suggestion
"/run/docker.sock:/var/run/docker.sock:rw
```